modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -663,6 +665,9 @@ private[internals] class Renderer(compilationUnit: CompilationUnit) { self =>
     )
   }
 
+  private def defaultThrowableGetMessage =
```

Comment:
**suggestion:** I would add a code comment with your explanation from the 0.18 PR:

> Default `toString` on `Throwable` prints the classname. This is suboptimal when we could have the case class `toString` implementation available.